packages/span-transformations/src/jaeger.ts

```diff
@@ -0,0 +1,70 @@
+import { SpanAttributes, SpanKind } from '@opentelemetry/api';
+import { timeInputToHrTime } from '@opentelemetry/core';
+import { ReadableSpan } from '@opentelemetry/sdk-trace-base';
+
+const getJaegerValueForTag = (jaegerTagKey, tags) => tags.filter(({ key }) => jaegerTagKey === key)?.[0]?.value;
```

Comment:
We strive to always state the parameters types, unless there is a good reason not to.